src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/Et3ResponseService.java

```diff
@@ -137,10 +142,31 @@ public void saveRelatedDocumentsToDocumentCollection(CaseData caseData) {
      * @param caseDetails Contains details about the case.
      */
     public void sendNotifications(CaseDetails caseDetails) {
-        emailService.sendEmail(et3EmailTribunalTemplateId,
+        emailService.sendEmail(
+            et3EmailTribunalTemplateId,
             caseDetails.getCaseData().getTribunalCorrespondenceEmail(),
             buildPersonalisation(caseDetails)
         );
     }
 
+    // todo group all personalisation methods and NotificationServiceConstants to EmailService or to a
+    // PersonalisationService. Might even be able to join into one single method that prepares all the data for all
+    // emails (unless some of it is expensive / bespoke / overrides in which case send a map parameter of overrides)
```

Comment:
Still relevant? I'm not sure about the idea of moving all personalisations into a single class. Sounds like the formation of a godclass that requires knowledge of too many things (like TornadoService). the personalisation objects are specific to each email template which is specific to the business needs of that service (ie ET3 form), it should remain up to ET3 related classes to know what data it needs for the template

Reply:
All personalisations have a core set of keys and very few have anything that's unique to the specific email being sent. We can make a generic method that adds those core properties and accepts any more unique ones as a parameter. it's just a thought though and if you disagree, I don't feel too strongly about it, we can remove the todo.